### PR TITLE
[RU]: menu and why-cypress page are translated to russian

### PR DESCRIPTION
**Pull request by viraxslot**
Added translation to Russian:
- menu
- why-cypress.md

@jennifer-shehane I have a couple of questions :)
1. Please see this [issue](https://github.com/cypress-io/cypress-documentation/issues/2333). I suppose we need to change "eslint-plugin-cypress-dev" dependency to "@cypress/eslint-plugin-dev", because the first one is deprecated. See [in origin repo of this plugin](https://github.com/cypress-io/eslint-plugin-dev#usage). I've fixed locally, but I'm not 100% sure.

2. Despite of all passed e2e tests I have problem in the side menu. Could you please help me how to fix absence of sections selection? For portuguese everything is ok, but russian page has no green mark (please see screenshots below).

3. Yeah, and the side menu is not wide enough for "Who uses Cypress?" phrase in Russian. Could we do something with it?

image
image

### CODE REVIEW

File: source/ru/guides/overview/why-cypress.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+title: Почему Cypress?
+---
+
+{% note info %}
+# {% fa fa-graduation-cap %} Что вы узнаете
+
+- Что такое Cypress и почему вам нужно его использовать
```
Comment (bahmutov) [inline source/ru/guides/overview/why-cypress.md]:
```suggestion
- Что такое Cypress и почему Вам нужно его использовать
```

File: themes/cypress/languages/ru.yml [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+index:
+  get-started: Быстрый старт
+
+page:
+  contents: Оглавление
+  back-to-top: Наверх
+  improve: Улучшить этот документ
+  prev: Назад
+  next: Вперёд
+  last-updated: "Последнее обновление: %s"
```
Comment (bahmutov) [inline themes/cypress/languages/ru.yml]: Hmm, why "%s"?

Comment (viraxslot) [reply themes/cypress/languages/ru.yml]: I don't know, copied from here https://github.com/cypress-io/cypress-documentation/blob/b8a035ca4b4d1e9deaa7721216865330d7cccd96/themes/cypress/languages/en.yml#L18

Comment (viraxslot) [reply themes/cypress/languages/ru.yml]: @bahmutov it's used for format string, for example here:
https://github.com/cypress-io/cypress-documentation/blob/b8a035ca4b4d1e9deaa7721216865330d7cccd96/themes/cypress/layout/api-toc.swig#L22

Hexo documentation says that "There is support for the printf format in language files": https://hexo.io/docs/internationalization#Language-Files

So it allows to convert date of the last update to string and use in the template :)

Comment (bahmutov) [reply themes/cypress/languages/ru.yml]: interesting, I did not notice that this could be customized per language

**Pull request merged**